contracts/Market.sol

```diff
@@ -57,9 +57,66 @@ contract Market is PausableAccessPreset {
 
   /**
    * @notice Emitted on setting of `_priorityRestrictedThreshold`.
+   * @param threshold The updated threshold for priority restricted supply.
    */
   event PriorityRestrictedThresholdSet(uint256 threshold);
 
+  /**
+   * @notice Emitted on setting of `_noriFeeWalletAddress`.
+   * @param updatedWalletAddress The updated address of the Nori fee wallet.
+   */
+  event NoriFeeWalletAddressUpdated(address updatedWalletAddress);
+
+  /**
+   * @notice Emitted on setting of `_noriFeePercentage`.
+   * @param updatedFeePercentage The updated fee percentage for Nori.
+   */
+  event NoriFeePercentageUpdated(uint256 updatedFeePercentage);
+
+  /**
+   * @notice Emitted when adding a supplier to `_activeSupply`.
+   * @param addedSupplierAddress The supplier that was added.
+   * @param nextSupplierAddress The next of the supplier that was added, updated to point to `addedSupplierAddress` as previous.
+   * @param previousSupplierAddress The previous of the supplier that was added, updated to point to `addedSupplierAddress` as next.
+   */
+  event SupplierAdded(
+    address indexed addedSupplierAddress,
+    address indexed nextSupplierAddress,
+    address indexed previousSupplierAddress
+  );
+
+  /**
+   * @notice Emitted when removing a supplier from `_activeSupply`.
+   * @param removedSupplierAddress The supplier that was removed.
+   * @param nextSupplierAddress The next of the supplier that was removed, updated to point to `previousSupplierAddress` as previous.
+   * @param previousSupplierAddress The previous of the supplier that was removed, updated to point to `nextSupplierAddress` as next.
+   */
+  event SupplierRemoved(
+    address indexed removedSupplierAddress,
+    address indexed nextSupplierAddress,
+    address indexed previousSupplierAddress
+  );
+
+  /**
+   * @notice Emitted when a removal is added to `_activeSupply`.
+   * @param removalId The removal that was added.
+   * @param supplierAddress The address of the supplier for the removal.
+   */
+  event RemovalAdded(
+    uint256 indexed removalId,
```

Comment:
seems redundant to use `removalId` in an event called `RemovalAdded` , perhaps `id` would be better?